lib/xcodeproj/scheme/archive_action.rb

```diff
@@ -0,0 +1,40 @@
+module Xcodeproj
+  class XCScheme
+    class ArchiveAction < XMLElementWrapper
+      def initialize(node = nil)
+        create_xml_element_with_fallback(node, 'ArchiveAction') do
+          self.build_configuration = 'Release'
+          self.reveal_archive_in_organizer = true
+        end
+      end
+
+      def build_configuration
+        @xml_element.attributes['buildConfiguration']
+      end
```

Comment:
A few other classes define methods like these as well. How about some common superclass or, if a class isn’t possible for some reason, some modules that define common behaviour?